github.com/cockroachdb/cockroach/pkg/server: TestStartNodeWithLocality failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/4af2bb46f9f5a873ca5a24b525c3c61930e60499

Parameters:
```
COCKROACH_PROPOSER_EVALUATED_KV=true
TAGS=deadlock
GOFLAGS=
```

Stress build found a failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=121168&tab=buildLog

```
I170118 09:53:45.099945 428402 gossip/client.go:131  [n?] started gossip client to 127.0.0.1:49768
I170118 09:53:45.100164 428422 gossip/server.go:214  [n?] received initial cluster-verification connection from {tcp 127.0.0.1:49768}
I170118 09:53:45.103252 427818 storage/store.go:1250  [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I170118 09:53:45.105348 428402 gossip/client.go:136  [n1] closing client to node 1 (127.0.0.1:49768): stopping outgoing client to node 1 (127.0.0.1:49768); loopback connection
F170118 09:53:45.105481 428391 gossip/node_set.go:140  nodeSet.placeholders should never be less than 0; gossip logic is broken &{nodes:map[0:{}] placeholders:-1 maxSize:3 gauge:0xc427fad8b0}
goroutine 428391 [running]:
github.com/cockroachdb/cockroach/pkg/util/log.getStacks(0x68901, 0x649854f, 0x28beac0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:849 +0xa5
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).outputLogEntry(0x28c0080, 0xc400000004, 0x20d6560, 0x12, 0x8c, 0xc423b22e10, 0x88)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:714 +0x784
github.com/cockroachdb/cockroach/pkg/util/log.addStructured(0x2b8d7e1191c0, 0xc4200146a0, 0x1b00000004, 0x2, 0x18ea81f, 0x4c, 0xc42c0e3c38, 0x1, 0x1)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/structured.go:146 +0x296
github.com/cockroachdb/cockroach/pkg/util/log.logDepth(0x2b8d7e1191c0, 0xc4200146a0, 0x1, 0xc400000004, 0x18ea81f, 0x4c, 0xc42c0e3c38, 0x1, 0x1)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/log.go:88 +0x8c
github.com/cockroachdb/cockroach/pkg/util/log.Fatalf(0x2b8d7e1191c0, 0xc4200146a0, 0x18ea81f, 0x4c, 0xc42c0e3c38, 0x1, 0x1)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/log.go:172 +0x7e
github.com/cockroachdb/cockroach/pkg/gossip.(*nodeSet).updateGauge(0xc4201ed3a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/node_set.go:140 +0x29e
github.com/cockroachdb/cockroach/pkg/gossip.(*nodeSet).removeNode(0xc4201ed3a0, 0xc400000001)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/node_set.go:119 +0x68
github.com/cockroachdb/cockroach/pkg/gossip.(*Gossip).removeClientLocked(0xc4201ed380, 0xc4255d0840)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/gossip.go:1254 +0x37e
github.com/cockroachdb/cockroach/pkg/gossip.(*Gossip).doDisconnected(0xc4201ed380, 0xc4255d0840)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/gossip.go:1153 +0x7f
github.com/cockroachdb/cockroach/pkg/gossip.(*Gossip).manage.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/gossip.go:1079 +0x26a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc427d64fa0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 1 [chan receive]:
testing.(*T).Run(0xc4200840c0, 0x18ae0d9, 0x19, 0x1a80128, 0xc42025dc01)
	/usr/local/go/src/testing/testing.go:647 +0x316
testing.RunTests.func1(0xc4200840c0)
	/usr/local/go/src/testing/testing.go:793 +0x6d
testing.tRunner(0xc4200840c0, 0xc42034ee00)
	/usr/local/go/src/testing/testing.go:610 +0x81
testing.RunTests(0x1a82b10, 0x23cffe0, 0x39, 0x39, 0x28fb1c0)
	/usr/local/go/src/testing/testing.go:799 +0x2f5
testing.(*M).Run(0xc42034eef8, 0x28face0)
	/usr/local/go/src/testing/testing.go:743 +0x85
github.com/cockroachdb/cockroach/pkg/server_test.TestMain(0xc42034eef8)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/main_test.go:32 +0x92
main.main()
	github.com/cockroachdb/cockroach/pkg/server/_test/_testmain.go:166 +0xc6

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2086 +0x1

goroutine 34 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).flushDaemon(0x28c0080)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1016 +0x77
created by github.com/cockroachdb/cockroach/pkg/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:581 +0x95

goroutine 428889 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*replicaScanner).waitAndProcess(0xc423a6f040, 0x2b8d7e1191c0, 0xc420014578, 0xed0113229, 0x642fa4f, 0x28beac0, 0xc425eb15c0, 0xc423ea53b0, 0xc42771af00, 0xed0113229)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scanner.go:207 +0x254
github.com/cockroachdb/cockroach/pkg/storage.(*replicaScanner).scanLoop.func1.1(0xc42771af00, 0xc42477fe30)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scanner.go:269 +0x8e
github.com/cockroachdb/cockroach/pkg/storage.(*storeReplicaVisitor).Visit(0xc42477fe30, 0xc423d093b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:345 +0x303
github.com/cockroachdb/cockroach/pkg/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scanner.go:271 +0x2ca
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc42225e440)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 427821 [chan receive]:
github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open.func1(0xc423715770)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:375 +0x44
created by github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:376 +0x702

goroutine 428386 [IO wait]:
net.runtime_pollWait(0x2b8d7e1e1838, 0x72, 0x0)
	/usr/local/go/src/runtime/netpoll.go:160 +0x59
net.(*pollDesc).wait(0xc423e0b170, 0x72, 0xc42559ed28, 0xc420014230)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
net.(*pollDesc).waitRead(0xc423e0b170, 0x23e0140, 0xc420014230)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
net.(*netFD).accept(0xc423e0b110, 0x0, 0x23dd400, 0xc425920760)
	/usr/local/go/src/net/fd_unix.go:419 +0x238
net.(*TCPListener).accept(0xc4286c3188, 0x6208de, 0xc42559edd8, 0x78ff1c)
	/usr/local/go/src/net/tcpsock_posix.go:132 +0x2e
net.(*TCPListener).Accept(0xc4286c3188, 0x1a82830, 0xc427965440, 0x23f66a0, 0xc42928cab0)
	/usr/local/go/src/net/tcpsock.go:222 +0x49
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).Serve(0xc427965440, 0x23ec640, 0xc4286c3188, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:369 +0x187
github.com/cockroachdb/cockroach/pkg/util/netutil.ListenAndServeGRPC.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:56 +0x3c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc423439a40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428655 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startGossip.func3()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1314 +0x39b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc425e3a9c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428336 [chan receive]:
github.com/cockroachdb/cockroach/pkg/rpc.NewContext.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:136 +0x76
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc4234392a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428627 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe00000002)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422d91f40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428933 [semacquire]:
sync.runtime_Semacquire(0xc420943554)
	/usr/local/go/src/runtime/sema.go:47 +0x30
sync.(*Mutex).Lock(0xc420943550)
	/usr/local/go/src/sync/mutex.go:85 +0xd0
sync.(*Mutex).Lock-fm()
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:55 +0x2a
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.lock.func1(0xc427dbc4f0, 0xc423feb260)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:151 +0x27
created by github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.lock
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:153 +0xf3

goroutine 428629 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe00000005)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422d91f80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428636 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe0000000c)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc4227240a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428410 [select]:
github.com/cockroachdb/cockroach/pkg/rpc.(*Context).runHeartbeat(0xc424282200, 0xc42b0b4dc0, 0xc427d64f10, 0xf, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:291 +0x629
github.com/cockroachdb/cockroach/pkg/rpc.(*Context).GRPCDial.func1.1.1()
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:226 +0x69
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc427ded050)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428632 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe00000001)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422d91fe0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428883 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:476 +0x383
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc42225e360)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428934 [semacquire]:
sync.runtime_Semacquire(0xc420943554)
	/usr/local/go/src/runtime/sema.go:47 +0x30
sync.(*Mutex).Lock(0xc420943550)
	/usr/local/go/src/sync/mutex.go:85 +0xd0
sync.(*Mutex).Lock-fm()
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:55 +0x2a
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.lock.func1(0xc427dbc530, 0xc423feb320)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:151 +0x27
created by github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.lock
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:153 +0xf3

goroutine 428639 [semacquire]:
sync.runtime_Semacquire(0xc4261d11d4)
	/usr/local/go/src/runtime/sema.go:47 +0x30
sync.(*WaitGroup).Wait(0xc4261d11c8)
	/usr/local/go/src/sync/waitgroup.go:131 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Wait(0xc4261d1130)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:187 +0x34
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Wait-fm()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3402 +0x2a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc423a3ec80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428656 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startGossip.func3()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1314 +0x39b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc425e3aa80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428413 [IO wait]:
net.runtime_pollWait(0x2b8d7e1e16b8, 0x72, 0x7)
	/usr/local/go/src/runtime/netpoll.go:160 +0x59
net.(*pollDesc).wait(0xc4235181b0, 0x72, 0xc42ab8b5f0, 0xc420014230)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
net.(*pollDesc).waitRead(0xc4235181b0, 0x23e0140, 0xc420014230)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
net.(*netFD).Read(0xc423518150, 0xc42a8f6000, 0x1000, 0x1000, 0x0, 0x23e0140, 0xc420014230)
	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
net.(*conn).Read(0xc42928cab0, 0xc42a8f6000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:173 +0x70
crypto/tls.(*block).readFromUntil(0xc427ded3b0, 0x2b8d7e09dd38, 0xc42928cab0, 0x5, 0xc42928cab0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:476 +0x91
crypto/tls.(*Conn).readRecord(0xc42baa3500, 0x1a83417, 0xc42baa3608, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:578 +0xc4
crypto/tls.(*Conn).Read(0xc42baa3500, 0xc4278ea000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1113 +0x116
bufio.(*Reader).fill(0xc427aa3b60)
	/usr/local/go/src/bufio/bufio.go:97 +0x10c
bufio.(*Reader).Read(0xc427aa3b60, 0xc42239fca8, 0x9, 0x9, 0x0, 0x63f6e0, 0x63f5a0)
	/usr/local/go/src/bufio/bufio.go:209 +0x1bc
io.ReadAtLeast(0x23d7a00, 0xc427aa3b60, 0xc42239fca8, 0x9, 0x9, 0x9, 0xc42ab8bb30, 0xf940ba, 0xc427ad0cf8)
	/usr/local/go/src/io/io.go:307 +0xa4
io.ReadFull(0x23d7a00, 0xc427aa3b60, 0xc42239fca8, 0x9, 0x9, 0x5fcbf8, 0x30, 0xc427ad0cf8)
	/usr/local/go/src/io/io.go:325 +0x58
github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.readFrameHeader(0xc42239fca8, 0x9, 0x9, 0x23d7a00, 0xc427aa3b60, 0x0, 0xc400000000, 0xc42a8de000, 0xd1a)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:237 +0x7b
github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.(*Framer).ReadFrame(0xc42239fc70, 0xd1a, 0xd1a, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:469 +0x8e
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*framer).readFrame(0xc425f7b050, 0xc425dae960, 0xc425dae960, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http_util.go:508 +0x2f
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Server).HandleStreams(0xc42239fe10, 0xc425f7b0e0, 0x1a82868)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:304 +0x201
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).serveStreams(0xc427965440, 0x23f50e0, 0xc42239fe10)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:477 +0x16c
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).serveHTTP2Transport(0xc427965440, 0x23f5500, 0xc42baa3500, 0x23df380, 0xc427a8eb00)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:458 +0x416
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).handleRawConn(0xc427965440, 0x23f66a0, 0xc42928cab0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:430 +0x470
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).Serve
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:399 +0x5ac

goroutine 428638 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe0000000b)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc4227240e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428886 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:476 +0x383
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc42225e3e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428641 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startCoalescedHeartbeatsLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3450 +0x167
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc423a3eca0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428419 [select]:
github.com/cockroachdb/cockroach/pkg/gossip.(*server).Gossip(0xc420943520, 0x23f55c0, 0xc427aa8b40, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/server.go:180 +0x50c
github.com/cockroachdb/cockroach/pkg/gossip._Gossip_Gossip_Handler(0x184c920, 0xc420943520, 0x23f3c40, 0xc423e4e480, 0xb, 0xc427aa8930)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/gossip.pb.go:209 +0xbb
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).processStreamingRPC(0xc427965440, 0x23f50e0, 0xc42239fe10, 0xc424ea4500, 0xc42477f0b0, 0x23c1000, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:807 +0x7d6
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).handleStream(0xc427965440, 0x23f50e0, 0xc42239fe10, 0xc424ea4500, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:897 +0xc36
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc425275e60, 0xc427965440, 0x23f50e0, 0xc42239fe10, 0xc424ea4500)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:469 +0xab
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).serveStreams.func1
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:470 +0xa3

goroutine 428653 [runnable]:
sync.runtime_Semacquire(0xc420189204)
	/usr/local/go/src/runtime/sema.go:47 +0x30
sync.(*Mutex).Lock(0xc420189200)
	/usr/local/go/src/sync/mutex.go:85 +0xd0
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.(*lockOrder).PostLock(0xc420189200, 0x4, 0x1730820, 0xc423a3e2d0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:232 +0x71
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.PostLock(0x4, 0x1730820, 0xc423a3e2d0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:133 +0x4b
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.lock(0xc427dbc560, 0x1730820, 0xc423a3e2d0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:191 +0xc36
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.(*Mutex).Lock(0xc423a3e2d0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:55 +0x78
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*TimedMutex).Lock(0xc4242557d8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/timedmutex.go:86 +0x34
github.com/cockroachdb/cockroach/pkg/storage.(*storeReplicaVisitor).Visit(0xc425daea80, 0xc4295d2e70)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:318 +0x4f
github.com/cockroachdb/cockroach/pkg/storage.(*Store).systemGossipUpdate(0xc424255500, 0xc4265ad800, 0x13, 0x20)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1386 +0xbe
github.com/cockroachdb/cockroach/pkg/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1223 +0xfc
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422724320)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428630 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe00000009)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422d91fa0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428625 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe00000004)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422d91f00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428637 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe00000003)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc4227240c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428407 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*addrConn).transportMonitor(0xc429ae82c0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/clientconn.go:819 +0x63e
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*ClientConn).resetAddrConn.func1(0xc429ae82c0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/clientconn.go:566 +0x1de
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*ClientConn).resetAddrConn
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/clientconn.go:567 +0x3c7

goroutine 428390 [select]:
github.com/cockroachdb/cockroach/pkg/gossip.(*Gossip).bootstrap.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/gossip.go:1037 +0x4c9
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc427d64f90)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428635 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe0000000e)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422724080)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428634 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe0000000f)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422724060)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428624 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe0000000d)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422d91ec0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428885 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:476 +0x383
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc42225e3c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428389 [chan receive]:
github.com/cockroachdb/cockroach/pkg/gossip.(*server).start.func3()
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/server.go:366 +0x66
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc423439b40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428626 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe00000008)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422d91f20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 423328 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Server).controller(0xc42239fe10)
	/go/src/github.com/cockroachdb/cockro129 runs completed, 1 failures, over 5m27s
ach/vendor/google.golang.org/grpc/transport/http2_server.go:727 +0x6ab
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Server
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:158 +0x8db

goroutine 428888 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:476 +0x383
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc42225e420)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428887 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:476 +0x383
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc42225e400)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428622 [chan receive]:
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:171 +0x55
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422d91e60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428654 [select]:
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.lock(0xc427dbc4f0, 0x1730820, 0xc420943550)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:157 +0xc5d
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.(*Mutex).Lock(0xc420943550)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:55 +0x78
github.com/cockroachdb/cockroach/pkg/gossip.(*Gossip).AddInfo(0xc4201ed380, 0x189b813, 0x8, 0xc42c0e9a58, 0x10, 0x10, 0x1bf08eb000, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/gossip.go:746 +0x50
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).gossipFirstRange(0xc42771af00, 0x2b8d7e0d7880, 0xc427ded6b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:4273 +0x29b
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).maybeGossipFirstRange(0xc42771af00, 0x2b8d7e0d7880, 0xc427ded6b0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:4258 +0x35d
github.com/cockroachdb/cockroach/pkg/storage.(*Store).maybeGossipFirstRange(0xc424255500, 0x2b8d7e0d7880, 0xc427ded6b0, 0x2b8d7e0d7880, 0xc427ded6b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1341 +0x1ac
github.com/cockroachdb/cockroach/pkg/storage.(*Store).(github.com/cockroachdb/cockroach/pkg/storage.maybeGossipFirstRange)-fm(0x2b8d7e0d7880, 0xc427ded6b0, 0xc420014578, 0x2b8d7e0d7880)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1279 +0x3e
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startGossip.func3()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1307 +0xf4
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc425e3a960)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 423329 [IO wait]:
net.runtime_pollWait(0x2b8d7e0ddf80, 0x72, 0x6)
	/usr/local/go/src/runtime/netpoll.go:160 +0x59
net.(*pollDesc).wait(0xc424c39fe0, 0x72, 0xc4255a3840, 0xc420014230)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x38
net.(*pollDesc).waitRead(0xc424c39fe0, 0x23e0140, 0xc420014230)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x34
net.(*netFD).Read(0xc424c39f80, 0xc428549000, 0x1000, 0x1000, 0x0, 0x23e0140, 0xc420014230)
	/usr/local/go/src/net/fd_unix.go:243 +0x1a1
net.(*conn).Read(0xc420031410, 0xc428549000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:173 +0x70
crypto/tls.(*block).readFromUntil(0xc42588aae0, 0x2b8d7e09dd38, 0xc420031410, 0x5, 0xc420031410, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:476 +0x91
crypto/tls.(*Conn).readRecord(0xc427670000, 0x1a83417, 0xc427670108, 0x5f0e0e)
	/usr/local/go/src/crypto/tls/conn.go:578 +0xc4
crypto/tls.(*Conn).Read(0xc427670000, 0xc429480000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1113 +0x116
bufio.(*Reader).fill(0xc427aa3e60)
	/usr/local/go/src/bufio/bufio.go:97 +0x10c
bufio.(*Reader).Read(0xc427aa3e60, 0xc42239ff18, 0x9, 0x9, 0xf940ba, 0xc425f7b298, 0xc425c50060)
	/usr/local/go/src/bufio/bufio.go:209 +0x1bc
io.ReadAtLeast(0x23d7a00, 0xc427aa3e60, 0xc42239ff18, 0x9, 0x9, 0x9, 0xc4255a3d38, 0xf95191, 0xc425f7b290)
	/usr/local/go/src/io/io.go:307 +0xa4
io.ReadFull(0x23d7a00, 0xc427aa3e60, 0xc42239ff18, 0x9, 0x9, 0xd08, 0xd08, 0x0)
	/usr/local/go/src/io/io.go:325 +0x58
github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.readFrameHeader(0xc42239ff18, 0x9, 0x9, 0x23d7a00, 0xc427aa3e60, 0x0, 0x0, 0xd08, 0xc42c387700)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:237 +0x7b
github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.(*Framer).ReadFrame(0xc42239fee0, 0x23e2600, 0xc427ded710, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:469 +0x8e
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*framer).readFrame(0xc425f7b1d0, 0xc427ded710, 0xc427ded710, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http_util.go:508 +0x2f
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).reader(0xc423ee7400)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:991 +0xbc
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:225 +0xb68

goroutine 427818 [runnable]:
github.com/cockroachdb/cockroach/vendor/github.com/petermattis/goid.Get(0x4)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/petermattis/goid/goid_go1.6plus.go:50
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.(*lockOrder).PreLock(0xc420189200, 0x4, 0x1730820, 0xc423a3e2d0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:242 +0xa8
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.PreLock(0x4, 0x1730820, 0xc423a3e2d0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:129 +0x4b
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.lock(0xc423a3ef70, 0x1730820, 0xc423a3e2d0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:145 +0x93
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.(*Mutex).Lock(0xc423a3e2d0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:55 +0x78
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*TimedMutex).Lock(0xc4242557d8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/timedmutex.go:86 +0x34
github.com/cockroachdb/cockroach/pkg/storage.(*storeReplicaVisitor).Visit(0xc425e3aba0, 0xc429fed4c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:318 +0x4f
github.com/cockroachdb/cockroach/pkg/storage.(*Store).LeaseCount(0xc424255500, 0x1)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:2318 +0xe4
github.com/cockroachdb/cockroach/pkg/storage.(*Store).Capacity(0xc424255500, 0xc429fed720, 0x10, 0xc423ea5300, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:2225 +0xcb
github.com/cockroachdb/cockroach/pkg/server.(*Node).initStores(0xc423e75180, 0x2b8d7e1191c0, 0xc420014578, 0xc427d648e0, 0x1, 0x1, 0xc423ea53b0, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:435 +0x4d8
github.com/cockroachdb/cockroach/pkg/server.(*Node).start(0xc423e75180, 0x2b8d7e1191c0, 0xc420014578, 0x23e2e00, 0xc42477f8c0, 0xc427d648e0, 0x1, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:346 +0x11d
github.com/cockroachdb/cockroach/pkg/server.createAndStartTestNode(0x23e2e00, 0xc42477f8c0, 0xc427d648e0, 0x1, 0x1, 0x23e20c0, 0xc4201dfe20, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node_test.go:130 +0x1c0
github.com/cockroachdb/cockroach/pkg/server.TestStartNodeWithLocality.func1(0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node_test.go:674 +0x26d
github.com/cockroachdb/cockroach/pkg/server.TestStartNodeWithLocality(0xc427964540)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node_test.go:716 +0x196
testing.tRunner(0xc427964540, 0x1a80128)
	/usr/local/go/src/testing/testing.go:610 +0x81
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:646 +0x2ec

goroutine 428422 [select]:
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.lock(0xc427dbc530, 0x1730820, 0xc420943550)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:157 +0xc5d
github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock.(*Mutex).Lock(0xc420943550)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/sasha-s/go-deadlock/deadlock.go:55 +0x78
github.com/cockroachdb/cockroach/pkg/gossip.(*server).gossipReceiver(0xc420943520, 0x2b8d7e0d78c0, 0xc426236b80, 0xc420114b08, 0xc426236bc0, 0xc423a71f40, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/server.go:322 +0x409
github.com/cockroachdb/cockroach/pkg/gossip.(*server).Gossip.func3.1()
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/server.go:143 +0x99
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc423d08550)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428882 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:476 +0x383
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc42225e340)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428623 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xc400000010)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422d91ea0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428628 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe00000007)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422d91f60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428631 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe00000006)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422d91fc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428381 [select]:
github.com/cockroachdb/cockroach/pkg/kv.(*TxnCoordSender).printStatsLoop(0xc423a6efa0, 0x2b8d7e1191c0, 0xc420014578)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/txn_coord_sender.go:209 +0xbd3
github.com/cockroachdb/cockroach/pkg/kv.NewTxnCoordSender.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/txn_coord_sender.go:193 +0x98
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc427d64db0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428385 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/netutil.ListenAndServeGRPC.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:49 +0x6f
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc42477f8f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428884 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:476 +0x383
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc42225e3a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428633 [semacquire]:
sync.runtime_notifyListWait(0xc425d14e50, 0xe0000000a)
	/usr/local/go/src/runtime/sema.go:267 +0x122
sync.(*Cond).Wait(0xc425d14e40)
	/usr/local/go/src/sync/cond.go:57 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4261d1130, 0xc423ea53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:212 +0xb2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:181 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc422724020)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66

goroutine 428434 [runnable]:
syscall.Syscall(0x1, 0x6, 0xc42a915000, 0x2a, 0x2a, 0x2a, 0x0)
	/usr/local/go/src/syscall/asm_linux_amd64.s:18 +0x5
syscall.write(0x6, 0xc42a915000, 0x2a, 0x1000, 0x77, 0x0, 0x0)
	/usr/local/go/src/syscall/zsyscall_linux_amd64.go:1064 +0x55
syscall.Write(0x6, 0xc42a915000, 0x2a, 0x1000, 0xd, 0xd, 0xc42a91500d)
	/usr/local/go/src/syscall/syscall_unix.go:180 +0x49
net.(*netFD).Write(0xc424c39f80, 0xc42a915000, 0x2a, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/fd_unix.go:321 +0x12a
net.(*conn).Write(0xc420031410, 0xc42a915000, 0x2a, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:185 +0x70
crypto/tls.(*Conn).write(0xc427670000, 0xc42a915000, 0x2a, 0x1000, 0xc42559fc01, 0xe263f5, 0xc42559fca0)
	/usr/local/go/src/crypto/tls/conn.go:819 +0x197
crypto/tls.(*Conn).writeRecordLocked(0xc427670000, 0x1a83417, 0xc42ac88000, 0xd, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:895 +0x343
crypto/tls.(*Conn).Write(0xc427670000, 0xc42ac88000, 0xd, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1047 +0x1fb
bufio.(*Writer).flush(0xc4240b0ac0, 0xc42559fdf8, 0xc42239ff60)
	/usr/local/go/src/bufio/bufio.go:563 +0x75
bufio.(*Writer).Flush(0xc4240b0ac0, 0x800000001, 0x0)
	/usr/local/go/src/bufio/bufio.go:552 +0x2b
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*framer).writeRSTStream(0xc425f7b1d0, 0x127aa3d01, 0x8, 0x1, 0x2b8d00000002)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http_util.go:468 +0x6d
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).controller(0xc423ee7400)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1085 +0x48f
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:255 +0x1135

goroutine 428640 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).raftTickLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3419 +0x334
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ea53b0, 0xc423a3ec90)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:197 +0x66


ERROR: exit status 255

Makefile:147: recipe for target 'stress' failed
make: *** [stress] Error 1
```

--- a/pkg/gossip/client.go
+++ b/pkg/gossip/client.go
@@ -233,11 +233,16 @@ func (c *client) handleResponse(ctx context.Context, g *Gossip, reply *Response)
 		g.maybeTightenLocked()
 	}
 	c.peerID = reply.NodeID
-	if !c.resolvedPlaceholder {
+	c.remoteHighWaterStamps = reply.HighWaterStamps
+
+	// If we haven't yet recorded which node ID we're connected to in the outgoing
+	// nodeSet, do so now. Note that we only want to do this if the peer has a
+	// node ID allocated (i.e. if it's nonzero), because otherwise it could change
+	// after we record it.
+	if !c.resolvedPlaceholder && c.peerID != 0 {
 		c.resolvedPlaceholder = true
 		g.outgoing.resolvePlaceholder(c.peerID)
 	}
-	c.remoteHighWaterStamps = reply.HighWaterStamps
 
 	// Handle remote forwarding.
 	if reply.AlternateAddr != nil {

--- a/pkg/server/node_test.go
+++ b/pkg/server/node_test.go
@@ -657,7 +657,7 @@ func TestStatusSummaries(t *testing.T) {
 func TestStartNodeWithLocality(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 
-	testLocalityWitNewNode := func(locality roachpb.Locality) {
+	testLocalityWithNewNode := func(locality roachpb.Locality) {
 		e := engine.NewInMem(roachpb.Attributes{}, 1<<20)
 		defer e.Close()
 		if _, err := bootstrapCluster(
@@ -713,6 +713,6 @@ func TestStartNodeWithLocality(t *testing.T) {
 	}
 
 	for _, testCase := range testCases {
-		testLocalityWitNewNode(testCase)
+		testLocalityWithNewNode(testCase)
 	}
 }

